config/routes.rb

```diff
@@ -11,8 +11,15 @@
 
   # get "/users/:id", to: "users#show"
 
+  # resources :users, only: [:index, :show] do
+  #   resources :posts, only: [:index, :show]
+  # end
```

Comment:
- [ ] Please kindly remove all the commented-out code from line 6 to 16, this will make your work to be neat and to avoid confusion later on. ℹ️